lib/tasks/participant_outcomes.rake

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+namespace :participant_outcomes do
+  desc "Send participant outcomes to the qualified teachers API"
+  task :send_to_qualified_teachers_api, %i[batch_size] => :environment do |_task, args|
+    job_klass = ParticipantOutcomes::BatchSendLatestOutcomesJob
+    batch_size = args[:batch_size]&.to_i || job_klass::DEFAULT_BATCH_SIZE
```

Comment:
could we also allow the `REQUEUE_DELAY` to be passed in as an argument as well, to give us more flexibility when running?